src/osd/ECBackend.cc

```diff
@@ -1572,8 +1611,23 @@ void ECBackend::submit_transaction(
   if (client_op) {
     op->trace = client_op->pg_trace;
   }
+  op->plan = ECTransaction::get_write_plan(
```

Comment:
For symmetry, seems like get_write_plan should also be a an ECBackend::Op method.

Reply:
Yes.

Reply:
Added a `static` wrapper.